source/reference/configure-shell-settings.txt

```diff
@@ -176,3 +218,52 @@ API call.
 
    Setting "enableTelemetry" has been changed
 
+Redact Sensitive Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Compare the recalled history when ``redactHistory`` is set to
+``remove-redact`` or ``remove``. 
+
+Set ``redactHistory`` to ``remove-redact`` mode and enter a query
+containing an email address. 
+
+.. code-block:: javascript
+
+   config.set( "redactHistory", "remove-redact" )
+   db.contacts.find( {"email": "customer@clients.com" } )
+
+When you press the ``up arrow`` to replay the last command the email
+address is redacted. 
+
+.. code-block:: javascript
+   :copyable: false
+
+   db.contacts.find( {"email": "<email>" } )  // Redacted
+
+Set ``redactHistory`` to ``remove`` mode and enter a query containing
+an email address. 
+
+.. code-block:: javascript
+
+   config.set( "redactHistory", "remove" )
+   db.contacts.find( {"email": "customer@clients.com" } )
+
+When you press the ``up arrow`` to replay the last command the email
+address is present. 
+
+.. code-block:: javascript
+   :copyable: false
+
+   db.contacts.find( {"email": "customer@clients.com" } )
+
+The :ref:`shell history log <mdb-shell-command-history>` reflects the
+changes. (Note that this log stores the most recent input first.)
```

Comment:
I think it might be helpful to avoid referring to history “log” files here, because mongosh also maintains log files that are referred to by that name which also contain the command history, even if in a different format

Reply:
Updated,